### PR TITLE
update latestPatchOf25 and releasing.md

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM

Should we maybe get a release train template like the Alpakkas and Akka HTTP?

**Comment by johanandren** [comment]
Weird with the MiMa check failing only on 2.12 on `javadsl.JavaFlowSupport`-related things

**Comment by raboof** [comment]
> Should we maybe get a release train template like the Alpakkas and Akka HTTP?

I think that'd be good!

**Comment by patriknw** [comment]
Had to move the "old forgotten" mima filters from 2.5.22 to 2.5.x file. (don't know why, but it works)

Also reverted the blog part changes of the release notes since we haven't fully decided there yet.

File: akka-stream/src/main/mima-filters/2.5.x.backwards.excludes [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Java9 RS APIs and more that was missed in a few earlier versions
```
Comment (raboof) [inline akka-stream/src/main/mima-filters/2.5.x.backwards.excludes]: Weird... ok?

**Review by raboof — APPROVED** [review]

File: akka-stream/src/main/mima-filters/2.5.x.backwards.excludes (hunk 1 above)
Comment (patriknw) [reply akka-stream/src/main/mima-filters/2.5.x.backwards.excludes]: yeah, I don't fully understand how these files are used

Comment (patriknw) [reply akka-stream/src/main/mima-filters/2.5.x.backwards.excludes]: somewhat similar thing in master https://github.com/akka/akka/pull/26995

**Comment by patriknw** [comment]
Added:

* Notify eng-updates
* Update Lightbend's supported versions

File: RELEASING.md [hunk 2]
```diff
@@ -52,23 +52,29 @@ a snapshot to https://repo.akka.io/snapshots from any branch.
...
 
-* Merge the release announcement
+* Publish the release announcement
 * Tweet about it
-* Post about it on Gitter and Discuss
+* Post about it on Discuss
+* Post about it on Gitter
+
+* Notify eng-updates
+* Update Lightbend's supported versions
```
Comment (patriknw) [inline RELEASING.md]: ah, this was mentioned below

**Pull request merged**